### DISCUSSION TITLE
hotkey for surface move

### DISCUSSION THREAD

**Discussion by sooooosh** [diesel-modding/BeardLib-Editor#718] [Implemented Ideas]
I've been using surface move a lot recently but going to the options every time to toggle it on or off gets a bit annoying.
A customizable hotkey to toggle it with a popup text would be really helpful imo.
![k67U9lE](https://user-images.githubusercontent.com/37846872/159853437-0b307083-4839-4e1c-bffd-dc6244b7bedd.png)

**Comment by Luffyyy** [comment]
iirc, you can turn on surface move by holding ctrl at the moment

**Comment by RexTheWho** [comment]
Surface Move is the one that aligns to the collision normal.
